manage

```diff
@@ -698,14 +712,32 @@ themes.all() {
     dump_return $?
 }
 
+themes.live() {
+    local theme
+    case "$1" in
+        simple|oscar)
+            theme=searx/static/themes/$1
+            ;;
+        *)
+            die 42 "missing theme argument"
```

Comment:
That error message also comes when misspelling a theme name in which case "missing argument" is misleading.

Also in the case of an error the error message is printed but since `themes.live` runs in a background job the `die` doesn't terminate the server ... which is a bit suboptimal since developers might miss the error message because with the default DEBUG startup output it will probably scroll of screen.

Reply:
@not-my-profile thanks for your helping remarks ..

> That error message also comes when misspelling a theme name

I changed the argument handling ...

```
$ ./manage themes.live xyz
ERROR: ./manage: line 792: themes.live(): unknown theme 'xyz' // [simple|oscar]

$ ./manage themes.live 
ERROR: ./manage: line 792: themes.live(): missing theme argument
```

> Also in the case of an error the error message is printed but since `themes.live` runs in a background job the `die` doesn't terminate the server 

I rearranged the processes and kill the process group if `themes.live` dies

```
$ LIVE_THEME=xyz make run
INFO    searx                         : load the default settings from /x/share/searxng/searx/settings.yml
INFO    searx                         : max_request_timeout=None
...
ERROR: ./manage: line 124: themes.live(): unknown theme 'xyz' // [simple|oscar]
make: *** [Makefile:28: run] Terminated
$
```

Reply:
@not-my-profile is the issue you named solved by my solution https://github.com/searxng/searxng/pull/664#discussion_r776499181?

Reply:
Yes, thanks!